BinaryOptionsToolsV2/BinaryOptionsToolsV2/pocketoption/asyncronous.py

```
@@ -6,23 +6,26 @@
 class AsyncSubscription:
     def __init__(self, subscription):
         """Asyncronous Iterator over json objects"""
         self.subscription = subscription
-        
+
     def __aiter__(self):
         return self
-        
+
     async def __anext__(self):
         return json.loads(await anext(self.subscription))
```

Ruff (0.13.1)

Undefined name `anext`. Consider specifying `requires-python = ">= 3.10"` or `tool.ruff.target-version = "py310"` in your `pyproject.toml` file.

(F821)